install.sh

```diff
@@ -68,9 +68,9 @@ have_sudo_access() {
 
   if [[ -z "${HAVE_SUDO_ACCESS-}" ]]; then
     if [[ -n "${args[*]-}" ]]; then
-      /usr/bin/sudo "${args[@]}" -l mkdir &>/dev/null
-    else
-      /usr/bin/sudo -l mkdir &>/dev/null
+      /usr/bin/sudo "${args[@]}" -v && /usr/bin/sudo "${args[@]}" -l mkdir &>/dev/null 
+  else
+      /usr/bin/sudo -v && /usr/bin/sudo -l mkdir &>/dev/null 
```

Comment:
```suggestion
      /usr/bin/sudo -v && /usr/bin/sudo -l mkdir &>/dev/null
```
Remove trailing white space.